app/models/setting/rh_cloud.rb

```diff
@@ -1,11 +1,20 @@
 class Setting::RhCloud < Setting
   ::Setting::BLANK_ATTRS.concat %w{rh_cloud_token}
 
+  def self.load_defaults
+    return false unless table_exists?
+    transaction do
+      # If the user had auto_upload default setting, we will not surprise him, and force the value to false
+      # for new users, the default will be set to true and the value will remain nil
+      Setting.where(name: 'allow_auto_inventory_upload', default: "--- false\n...", value: nil).update_all(value: "--- false\n...")
```

Comment:
I think I found the issue, the `\n` doesn't catch the default value:

```suggestion
      Setting.where(name: 'allow_auto_inventory_upload', default: "--- false", value: nil).update_all(value: "--- false\n...")
```

![Selection_002](https://user-images.githubusercontent.com/26363699/107922012-0e344f80-6f78-11eb-9a6e-14f794abe9d7.png)

Btw, can't the default value be just `false`? why are the `---` additions needed?

Reply:
We're saving the values as yaml-encoded. As for the `\n` - I think it's encoded with `\n`, take a look at the value after the change, if it's `true\n`, then it was `false\n` beforehand.

Reply:
@ShimShtein when switching with your PR, after seed to default `true` it also shows without the `/n`:
![Selection_003](https://user-images.githubusercontent.com/26363699/107928531-1e046180-6f81-11eb-9bb8-e65091da50de.png)

but looks like in older versions of Foreman, like 2.1-stable, it has `default: "--- false\n..."` in it, @ezr-ondrej any idea? :D